basic/stam/stamExcersize/stam.py

```diff
@@ -0,0 +1,49 @@
+__author__ = 'Maya Gal-Yam'
+
+import sys
+import os
+from os import path
+
+ORIGINAL_FILE_LOCATION = 1
+NEW_LINE_FORMAT = 'Maya {} Gal-Yam\n'
+NEW_FILE_NAME = 'newfile.txt'
+
+
+def get_file():
+    """gets the files passed in if args are valid"""
+    try:
+        originalfile_path = sys.argv[ORIGINAL_FILE_LOCATION]
+        if path.exists(originalfile_path) and path.isfile(originalfile_path):
```

Comment:
`path.exists` is redundant, `path.isfile` also checks if the file exists

Also, the whole checking is redundant, as the same thing will happen in`open` (but with fewer lines)